modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -78,7 +78,8 @@ CREATE TABLE IDN_OAUTH2_ACCESS_TOKEN (
             TOKEN_SCOPE_HASH VARCHAR2 (32),
             TOKEN_STATE VARCHAR2 (25) DEFAULT 'ACTIVE',
             TOKEN_STATE_ID VARCHAR (128) DEFAULT 'NONE',
-            PRIMARY KEY (TOKEN_ID),
+            SUBJECT_IDENTIFIER VARCHAR2(255),
+	    PRIMARY KEY (TOKEN_ID),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04